boards/arm/stm32h747i_disco/stm32h747i_disco_m7.dts

```diff
@@ -54,3 +59,87 @@
 	current-speed = <115200>;
 	status = "okay";
 };
+
+&fmc {
+	status = "okay";
+	pinctrl-0 = <&fmc_nbl0_pe0
```

Comment:
I would prefer a list in the following format:
```
pinctrl-0 = <&fmc_nbl0_pe0  &fmc_nbl1_pe1 &fmc_nbl2_pi4
		     &fmc_nbl3_pi5  &fmc_sdclk_pg8 &fmc_sdnwe_ph5
		     &fmc_sdcke1_ph7 &fmc_sdne1_ph6 &fmc_sdnras_pf11 
		     &fmc_sdncas_pg1 ....
```
but that's a pure matter of taste

Reply:
done